app/ceom/aboutus/urls.py

```diff
@@ -23,4 +23,6 @@
     re_path(r'^group_photos/(?P<selYear>\d{4})', ceom.aboutus.views.group_photos),
     re_path(r'^group_photos', ceom.aboutus.views.group_photos),
     re_path(r'$', ceom.aboutus.views.news),
+    re_path(r'^stats/user/',ceom.aboutus.views.user_stats),
```

Comment:
For the URL, having the extra slash in the middle will work just fine, but usually this would probably be something like
`stats_user/`
`stats_photo/`

The reason is that each level of / is meant to correspond with a single urls.py file
So `aboutus/` is in the main urls.py for the whole project 
And `aboutus/news/` is in specifically the aboutus urls.py
So `aboutus/stats/photos/` implies that it's in the aboutus-stats urls.py, but we don't have a file like that

The app as a whole doesn't always adhere to this rule, but that's usually the organization I try to keep